include/u-boot.mk

```diff
@@ -18,6 +20,27 @@ PKG_LICENSE_FILES:=Licenses/README
 
 PKG_BUILD_PARALLEL:=1
 
+ifdef UBOOT_USE_BINMAN
+  $(eval $(call TestHostCommand,python3-pyelftools, \
+    Please install the Python3 elftools module, \
+    $(STAGING_DIR_HOST)/bin/python3 -c 'import elftools'))
+endif
+
+ifdef UBOOT_USE_INTREE_DTC
+  $(eval $(call TestHostCommand,swig, \
```

Comment:
When are these checks done? 
Does it require swig only when a uboot with `UBOOT_USE_INTREE_DTC` is build?
I was looking into this some time ago and was unable to make it work like this.

Reply:
> When are these checks done?

Once the corresponding package is selected, when user executes `make` command. this check will be triggered.

> Does it require swig only when a uboot with `UBOOT_USE_INTREE_DTC` is build?

Yep.

> I was looking into this some time ago and was unable to make it work like this.

`prereq.mk` needs to be included explicitly, I guess you forgot it? ;)